web/src/components/table/data-table-pagination.tsx

```diff
@@ -59,7 +67,10 @@ export function DataTablePagination<TData>({
           <Button
             variant="outline"
             className="hidden h-8 w-8 p-0 lg:flex"
-            onClick={() => table.setPageIndex(0)}
+            onClick={() => {
+              table.setPageIndex(0);
+              posthog.capture("table:pagination_button_click", { table: tableName });
```

Comment:
Should we add a type for pagination_button_click event? E.g. "firstPage", "prevPage", "nextPage", "lastPage"